qadence/types.py

```diff
@@ -122,6 +124,26 @@ class LTSOrder(StrEnum):
     """ST4."""
 
 
+class BasisFeatureMap(StrEnum):
+    """Basis set for feature maps."""
+
+    FOURIER = "Fourier"
+    """Fourier basis set."""
+    CHEBYSHEV = "Chebyshev"
+    """Chebyshev polynomials of the first kind."""
+
+
+class ScalingFeatureMap(StrEnum):
```

Comment:
This should be renamed back to ReuploadScalingFeatureMap

Reply:
Thanks @vincentelfving. Maybe I would keep it just as ReuploadScaling as before just to keep it short, @Roland-djee ?